test_runner/fixtures/zenith_fixtures.py

```diff
@@ -277,29 +278,36 @@ def connstr(self,
         if schema:
             res = f"{res} options='-c search_path={schema}'"
 
+        if statement_timeout_ms:
+            res = f"{res} options='-c statement_timeout={statement_timeout_ms}'"
+
         return res
 
     # autocommit=True here by default because that's what we need most of the time
-    def connect(self,
-                *,
-                autocommit=True,
-                dbname: Optional[str] = None,
-                schema: Optional[str] = None,
-                username: Optional[str] = None,
-                password: Optional[str] = None) -> PgConnection:
+    def connect(
+        self,
+        *,
+        autocommit=True,
+        dbname: Optional[str] = None,
+        schema: Optional[str] = None,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        # individual statement timeout in seconds, 2 minutes should be enough for our tests
+        statement_timeout: Optional[int] = 120
```

Comment:
NIT: we can add `_s` or _`seconds` suffix here too/

Reply:
I thought of it, but `statement_timeout_s` doesn't look pretty IMO and `statement_timeout_seconds` is quite long. Decided to keep it that way because python use seconds as default time units in many places.